clean-abap/CleanABAP.md

```diff
@@ -924,9 +924,9 @@ ENDMETHOD.
 
 > Read more in _Chapter 5: Formatting: Vertical Distance: Variable Declarations_ of [Robert C. Martin's _Clean Code_].
 
-### Don't declare inline in optional branches
+### Do not use local variables outside of the branch they are declared in
```

Comment:
Nitpick: There are [control structures](https://help.sap.com/doc/abapdocu_cp_index_htm/CLOUD/en-US/index.htm?file=abencontrol_structure_glosry.htm) that are not branches (a `DO 2 TIMES.` loop, for example is not a branch - the control flow graph does not fork here). In the terminology of the ABAP Keyword Documentation, what we actually mean is that variables should not be used outside the [statement block](https://help.sap.com/doc/abapdocu_cp_index_htm/CLOUD/en-US/index.htm?file=abenstatement_block_glosry.htm) they're declared in. So 'branch' should be replaced by 'statement block' if we want to be accurate.

Reply:
yea, I was also thinking about using the word "block" instead

the word "branch" originates from the old text

Reply:
Sure, I'm all for using the correct terms. OK to resolve?